### ISSUE TITLE
Don't respond..

### ISSUE THREAD

**Issue by MariuszPL55** [gilgame/SEWorkbench#20]
Tool again after update even won't start.
Magic, now works and I do nothing..

**Comment by gilgame** [comment]
Did the program get to the prompt "SE Workbench needs to copy the sandbox files." beforehand?

**Comment by gilgame** [comment]
I ask because there have been reports of the program crashing on start, but I haven't been able to gather enough information to figure out why.

**Comment by mmille14** [comment]
I am crashing on start as well. The program prompted that it needed to copy the sandbox files than prompted me for the location of space engineers exe. After that, boom. 

Here are the details. 
Problem signature:
  Problem Event Name:   CLR20r3
  Problem Signature 01: Gilgame.SEWorkbench.exe
  Problem Signature 02: 0.0.0.0
  Problem Signature 03: 5706f4ed
  Problem Signature 04: Gilgame.SEWorkbench.Interop
  Problem Signature 05: 1.0.0.0
  Problem Signature 06: 5706f4e9
  Problem Signature 07: 32
  Problem Signature 08: 1c
  Problem Signature 09: System.MissingMethodException
  OS Version:   6.1.7601.2.1.0.256.48
  Locale ID:    1033
  Additional Information 1: 40cf
  Additional Information 2: 40cf079a96ef2995d5fbb3c34d123859
  Additional Information 3: fb51
  Additional Information 4: fb512bf96a4e23656fd0629c24d3ea8e

**Comment by MariuszPL55** [comment]
Yes, program prompt about copying files, and after that boom and crash.
